FreeRTOS_Sockets.c

```diff
@@ -913,6 +978,121 @@ Socket_t FreeRTOS_socket( BaseType_t xDomain,
 #endif /* ipconfigSUPPORT_SELECT_FUNCTION == 1 */
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief : called from FreeRTOS_recvfrom(). This function waits for an incoming
+ *          UDP packet, or until a time-out occurs.
+ * @param[in] pxSocket : The socket that receives UDP packets.
+ * @param[in] xFlags : The flags as passed to FreeRTOS_recvfrom().
+ * @param[in,out] pxEventBits : The last even received in this function,
+ *                              either eSOCKET_INTR or eSOCKET_RECEIVE.
+ */
+static NetworkBufferDescriptor_t * prvRecvFromWaitPacket( FreeRTOS_Socket_t const * pxSocket,
```

Comment:
The name is descriptive but not as clear as I would like I think. (I had to go to the place from where this is called and then I understood what this does :) )
How does the name `prvRecvFromWaitForPacket` sound? I just added a "For" in there.

Reply:
Yes I agree, "wait for packet" is clearer than "wait packet",